README.md

```diff
@@ -63,11 +63,11 @@ To build and test in Visual Studio, open
 Alternatively, you can build REEF.NET from a developer command line
 via:
 
-    msbuild .\lang\cs\Org.Apache.REEF.sln
+    msbuild .\lang\cs\Org.Apache.REEF.sln /p:Configuration="Debug" /p:Platform="x64"
```

Comment:
Why are these new flags needed? Wouldn't they be the default? I am asking because I think keeping the build instructions as simple as possible is a value in its own right :)